lib/protobuf/builder.ex

```diff
@@ -13,6 +13,9 @@ defmodule Protobuf.Builder do
     new_maybe_strict(mod, attrs, _strict? = true)
   end
 
+  def field_default(_, %{options: options} = props) when not is_nil(options) do
```

Comment:
What if options is an empty list? Should you handle in `Protobuf.FieldOptionsProcessor`?

Reply:
added